de.metas.business/src/main/java-legacy/org/compiere/model/CalloutOrder.java

```diff
@@ -342,6 +342,12 @@ public String bPartner(final ICalloutField calloutField)
 				+ " = stats."
 				+ I_C_BPartner_Stats.COLUMNNAME_C_BPartner_ID
 				+ ")"
+				+ " LEFT JOIN " + I_C_BPartner_CreditLimit.Table_Name
+				+ " cl ON (cl."
+				+ I_C_BPartner_CreditLimit.COLUMNNAME_C_BPartner_ID
+				+ " = stats."
+				+ I_C_BPartner_Stats.COLUMNNAME_C_BPartner_ID
+				+ ") "
```

Comment:
what would happen in case there are more than one C_BPartner_CreditLimit records for that BPartner?
why not using the java API which returns the credit limit for given BP?

Reply:
We have an order by clause, like for others in this sql.
And will take only the first result.
I did not used the API because I did not want to rewrite this.